src/gramian.jl

```diff
@@ -0,0 +1,111 @@
+"""
+    empirical_ctrl_gramian()
+
+A function for computation of controllability gramian.
+
+# Notes
+- pr = parameter vector(s), each column i sone parameter sample
+- xs = steady-state and nominal initial state x_0
+- us = steady-state input
+"""
+function empirical_ctrl_gramian(f, s; dt::Real, tf::Real, pr::Matrix, xs::Vector)
+    M, N, L = s
```

Comment:
```suggestion
function empirical_ctrl_gramian(f, M::Int, N::Int, L::Int; dt::Real, tf::Real, pr::Matrix, xs::Vector)
```

인자를 직접 받는게 좋을 것 같습니다 (`s` 가 특정되지 않아서)